src/main/java/step3/CarRacing.java

```diff
@@ -0,0 +1,52 @@
+package step3;
+
+import java.util.Random;
+
+public class CarRacing {
+	private static final int CAN_MOVE_NUM = 4;
+	private static final int RANDOM_LIMIT = 9;
+	private static final String MOVE_MARK = "-";
+	private static final Random random = new Random();
+
+	public static void main(String[] args) {
+		int carCount = InputView.requestHowManyCar();
+		int times = InputView.requestHowManyTimes();
+
+		StringBuilder[] cars = getNewCars(carCount);
+		StringBuilder[] raceResult = new StringBuilder[times];
+
+		for (int i = 0; i < times; i++) {
+			raceResult[i] = moveForward(cars);
+		}
+
+		ResultView.printResult(raceResult);
+	}
+
+	public static StringBuilder[] getNewCars(int carCount) {
+		StringBuilder[] cars = new StringBuilder[carCount];
+		for (int i = 0; i < cars.length; i++) {
+			cars[i] = new StringBuilder();
+		}
+		return cars;
+	}
+
+	private static StringBuilder moveForward(StringBuilder[] cars) {
+		StringBuilder roundResult = new StringBuilder();
+		for (int i=0; i < cars.length; i++) {
```

Comment:
단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다 :)
다른 부분은 잘 지켜주셨지만 이 부분만 안되어 있네요!
```suggestion
		for (int i = 0; i < cars.length; i++) {
```